data/_ui/tooltips.txt

```diff
@@ -373,28 +376,37 @@ tip "leak resistance heat:"
 	`Heat generated per second when resisting leak damage.`
 
 tip "maintenance costs:"
-	`How many credits per day you need to spend to maintain this outfit. Even if a ship is parked or an outfit is in cargo, you still need to pay for its maintenance.`
+	`Credits spent per day to maintain this outfit. Even if a ship is parked or an outfit is in cargo, you still need to pay for its maintenance.`
 
 tip "map:"
-	`How many star systems are included in this map.`
+	`The number of star systems included in this map.`
 
 tip "mass with no cargo:"
 	`Mass of this ship when not carrying any cargo or fighters. Cargo increases mass, which reduces acceleration and turn rate and increases maximum heat before overheating.`
 
 tip "mass:"
 	`Mass, in tons. A ship's mass determines how fast it turns and accelerates and its maximum heat before overheating.`
 
+tip "max speed:"
+	`The maximum speed of the ship when using thrusters but no afterburners.`
```

Comment:
Max speed actually can display the maximum speed while thrusters are active if there are no normal thrusters installed.

Also spotted that this says "the ship" while other tooltips around it say "this ship." Might be worth running through all the tooltips to standardize that. For these tooltips that only appear in the ship info sidebar, I think "this ship" makes sense.
```suggestion
	`The maximum speed of this ship when using thrusters. Excludes the effect of afterburners, unless there are no normal thrusters installed.`
```

Reply:
Both `the ship` and `this ship` are used quite often.